src/App.js

```diff
@@ -60,6 +60,11 @@ class App extends Component {
 
     }
 
+    componentDidUpdate() {
```

Comment:
We already have `componentDidUpdate` at line 41 and we are using `updateDocumentTitle`.

Reply:
[#L41-L52](https://github.com/RedHatInsights/ros-frontend/blob/main/src/App.js#L41-L52)

Reply:
@PreetiW  -  any updates here? Since long time, it is in open state.